fengfengzhidao_fengfeng_blog_api_views_image_51.py

```
@@ -0,0 +1,16 @@
+class ImageView(View):
+    """
+    图片上传
+    """
+
+    def post(self, request: WSGIRequest):
+        file = request.FILES.get('file')
+        file_name = f'media/uploads/{file.name}'
+        with open(file_name, 'wb+') as pic:
+            for c in file.chunks():
+                pic.write(c)
+        return ok(f"/{file_name}", "上传成功")
```

Ruff (0.13.1)

`ok` may be undefined, or defined from star imports

(F405)